media-picker/src/main/java/com/miguelgaeta/media_picker/MediaPicker.java

```diff
@@ -190,8 +190,8 @@ public static void startForImageCrop(final Provider provider, final Uri uri, int
                 ? options
                 : new UCrop.Options();
 
-            // Force JPEG format compression now while working out filename strategy.
-            cropOptions.setCompressionFormat(Bitmap.CompressFormat.JPEG);
+            // Force PNG format compression
+            cropOptions.setCompressionFormat(Bitmap.CompressFormat.PNG);
```

Comment:
can you explain why we need this change in the description?
I'm wondering why it changed from JPEG to PNG.

Reply:
JPEGs dont support transparency, right? This is the whole reason we are doing it.